tasks/sep/006-1-foundation-guardian-updates/SignFromJson.s.sol

```diff
@@ -0,0 +1,537 @@
+// SPDX-License-Identifier: MIT
```

Comment:
![Screenshot 2024-04-19 at 10 27 05](https://github.com/ethereum-optimism/superchain-ops/assets/15608778/56b02c00-4033-4b58-82e6-8ebb809827e6)
Getting an error when I simulate.

Reply:
yep, that's intentional.
What's happening is that this runbook assumes that the previous one has already been executed. Once it has, those errors will disappear.